inst/examples/06-publishing.Rmd

```diff
@@ -304,7 +304,7 @@ You *can* update this kind of site, but it is a manual update. In the Netlify UI
 
 ![Screenshot of drag and drop deploy **update** box on Netlify](images/netlify-drag-drop-update.png)
 
-Edit your book, build it locally again, then drag and drop the output directory here again. This workflow is great to quickly share a book prototype, but is not ideal for books you are actively editing or collaborating on. As you might expect, there is an easier way! We can connect GitHub to Netlify and use continuous deployment.
+Edit your book, build it locally again, then drag and drop the output directory here again. This workflow is great to quickly share a book prototype, but is not ideal for books you are actively editing or collaborating on. As you might expect, there is an easier long-term solution! We can connect GitHub to Netlify and use continuous deployment.
```

Comment:
Not sure if it is "easier" actually to some so I wanted to modify the words a bit.